src/main/java/com/cincialert/service/UserService.java

```diff
@@ -3,35 +3,30 @@
 import org.springframework.beans.factory.annotation.Autowired;
```

Comment:
The reason for this is because when you go to call the repository for the model, it will be listed as the model repository anyways, so you don't need to define what is being deleted or added in the naming convention.